ui/src/components/erc/erc.css

```diff
@@ -1,3 +1,11 @@
 .Erc {
   text-align: center;
 }
+
+#PreviewNotice{
+  color: white;
+  background-color: orange;
```

Comment:
We've created a little color palette way back, it might be worth to take an orange from it that works nicely with our "blue". See #192

Reply:
I have used the color[ #CE5100 ](https://www.color-hex.com/color/ce5100 )which is defined in the the palette for warnings.

Reply:
Good!